src/admin/src/routes/AppRouter.js

```diff
@@ -29,8 +29,8 @@ function AppRouter() {
           <Route exact path="/bcparks/cms-contents">
             <CmsContents />
           </Route>
-          <Route exact path="/bcparks/park-status">
-            <ParkStatus />
+          <Route exact path="/bcparks/park-access-status">
+            <ParkAccessStatus />
```

Comment:
this should be a private route for submitter and approver roles. we dont want unauthenticated users to access our data

Reply:
Hi Siva, I checked with Rob, he said this is open to all users.